playground/frontend/lib/pages/playground/components/feedback/feedback_dropdown_content.dart

```diff
@@ -0,0 +1,182 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter/material.dart';
+import 'package:playground/config/theme.dart';
+import 'package:playground/constants/font_weight.dart';
+import 'package:playground/constants/fonts.dart';
+import 'package:playground/constants/sizes.dart';
+import 'package:playground/modules/analytics/analytics_service.dart';
+
+import 'feedback_dropdown_icon_button.dart';
+
+const double kTextFieldWidth = 365.0;
+const double kTextFieldHeight = 68.0;
+const String kFeedbackTitleText = 'Feedback';
+const String kCancelButtonTitle = 'Cancel';
+const String kSendFeedbackButtonTitle = 'Send feedback';
+const String kFeedbackContentText =
+    'Tell us what you are missing and you would like to see it\n'
```

Comment:
GCP, Youtube and Keep all use this feedback text:
```
Have feedback? We'd love to hear it, but please don't share sensitive information. Have questions? Try help or support.
```

Reply:
@KonradJanica thank you for the review, i've changed the feedback text.